managed/src/main/java/com/yugabyte/yw/controllers/InstanceTypeController.java

```diff
@@ -26,13 +27,20 @@
 public class InstanceTypeController extends AuthenticatedController {
 
   public static final Logger LOG = LoggerFactory.getLogger(InstanceTypeController.class);
+  private final Config config;
 
   @Inject
   FormFactory formFactory;
 
   @Inject
   CloudAPI.Factory cloudAPIFactory;
 
+  // TODO: Remove this when we have HelperMethod in place to get Config details
+  @Inject
+  public InstanceTypeController(Config config) {
+    this.config = config;
```

Comment:
Do not mix ctor injections and field injections in same class.
Either fix all existing fields to be parameters here or make the new field use field injection.

FWIW, using ctor injection with private fina fields is the best practice. But i ll leave it to you which option you want for now.

Reply:
Done. Thanks!